### PR TITLE
Update new-relic-api-keys.mdx

### PR DESCRIPTION
**Pull request by johnathannewrelic**
TSE on the #gts-accounts-security team updating this doc to remove references to things that were removed from the product intentionally.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 1]
```diff
@@ -185,11 +174,11 @@ If your API keys get into the wrong hands, it can present a security risk. For e
 You should treat your API keys securely, as you would passwords and other sensitive information. Some recommendations:
 
 * For the license key and the browser key, consider implementing a key rotation strategy: creating new keys and deleting old ones on a set schedule. Considerations:
-  * You can't delete the original keys associated with an account, so to implement a strong security strategy, you must create additional keys that you can later delete.
+  * You can't delete the original ingest keys associated with an account, so to implement a strong security strategy, you must create additional ingest keys that you can later delete.
   * Note that this doesn't apply to the mobile app token; you can't delete a token or create additional tokens.
 * For the <InlinePopover type="userKey"/>:
   * Instruct your team members to keep their user keys secure.
-  * When members leave your organization, remove them from New Relic.
+  * When members leave your organization, remove them from New Relic which will disable all the user keys assocaited with their user ID
```
Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:181]:
```suggestion
  * When members leave your organization, remove them from New Relic which will disable all the user keys assocaited with their user ID.
```

Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:181]:
```suggestion
  * When members leave your organization, remove them from New Relic. This disables all the user keys associated with their user ID. 
```

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 2]
```diff
@@ -160,20 +160,9 @@ To get started with API keys:
...
-The [account ID](/docs/accounts/accounts-billing/account-setup/account-id) is another identifying number often required for reporting data to New Relic.
-
-## Copying keys from the UI [#copy-keys]
-
-You can copy existing API keys and their IDs from [<DNT>**one.newrelic.com/api-keys**</DNT>](https://one.newrelic.com/api-keys):
-
-* <DNT>**Copy key**</DNT>: Copies the value of the key itself.
-* <DNT>**Copy key ID**</DNT>: Copies the ID of the key, which is sometimes needed for referencing the key object via API.
-* <DNT>**Copy truncated key**</DNT>: This option is available for keys that are not yours. It copies only a few digits from the key, which can be useful for your internal tracking of keys, or for talking with support.
+To view a key you will need to copy the key ID which you will need to view the key via Nerdgraph, which you can also [use to manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys)
```
Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:165]:
```suggestion
To view a key in NerdGraph, copy the key ID.  You can also [use NerdGraph to manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys)
```

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 3]
```diff
@@ -185,11 +174,11 @@ If your API keys get into the wrong hands, it can present a security risk. For e
 You should treat your API keys securely, as you would passwords and other sensitive information. Some recommendations:
 
 * For the license key and the browser key, consider implementing a key rotation strategy: creating new keys and deleting old ones on a set schedule. Considerations:
-  * You can't delete the original keys associated with an account, so to implement a strong security strategy, you must create additional keys that you can later delete.
+  * You can't delete the original ingest keys associated with an account, so to implement a strong security strategy, you must create additional ingest keys that you can later delete.
```
Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:177]:
```suggestion
  * You can't delete the original ingest keys associated with an account, so we recommend creating additional ingest keys that you can later delete. This ensures you're assuming a strong security posture. 
```

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 4]
```diff
@@ -185,11 +174,11 @@ If your API keys get into the wrong hands, it can present a security risk. For e
...
   * Note that this doesn't apply to the mobile app token; you can't delete a token or create additional tokens.
 * For the <InlinePopover type="userKey"/>:
   * Instruct your team members to keep their user keys secure.
-  * When members leave your organization, remove them from New Relic.
+  * When members leave your organization, remove them from New Relic. This disables all the user keys associated with their user ID.
```
Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:181]:
```suggestion
  * When members leave your organization, remove their user IDs from New Relic. This disables all the user keys associated with their user ID. 
```

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx (hunk 1 above)
Comment (johnathannewrelic) [reply src/content/docs/apis/intro-apis/new-relic-api-keys.mdx]: This one looks good!

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 5]
```diff
@@ -160,20 +160,9 @@ To get started with API keys:
 
 ## View and manage API keys [#keys-ui]
 
-You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DNT>**API keys**</DNT>. You can also:
+You can manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DNT>**API keys**</DNT>.
```
[the remark this answers is not in the document]
Comment (johnathannewrelic) [reply src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:163]: This one looks good!

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx (hunk 2 above)
Comment (johnathannewrelic) [reply src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:165]: This looks good!

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx (hunk 3 above)
Comment (johnathannewrelic) [reply src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:177]: This looks good!

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx (hunk 4 above)
Comment (johnathannewrelic) [reply src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:181]: This looks good!

**Review by akristen — APPROVED** [review]

**Pull request merged**